web/docs/guides/auth-ui.md

```diff
@@ -0,0 +1,217 @@
+
+// Use it like this
+<LoginForm />
+```
+
+### Signup form
+
+Useful for <span style={{ background: '#fee', padding: '0.2rem 0.5rem', borderRadius: '0.375rem' }}>username & password</span> and <span style={{ background: '#eef', padding: '0.2rem 0.5rem', borderRadius: '0.375rem' }}>email</span> authentication.
+
+![Signup form](/img/authui/signup.png)
+
+You can use the `SignupForm` component to build your own signup form.
+
+```tsx title="client/SignupPage.tsx"
+import { SignupForm } from '@wasp/auth/forms/Signup'
+
+// Use it like this
+<SignupForm />
+```
+
+### Forgot password form
+
+Useful for <span style={{ background: '#eef', padding: '0.2rem 0.5rem', borderRadius: '0.375rem' }}>email</span> authentication.
+
+![Forgot password form](/img/authui/forgot_password.png)
+
+You can use the `ForgotPasswordForm` component to build your own forgot password form.
+
+```tsx title="client/ForgotPasswordPage.tsx"
+import { ForgotPasswordForm } from '@wasp/auth/forms/ForgotPassword'
+
+// Use it like this
+<ForgotPasswordForm />
+```
+
+### Reset password form
+
+Useful for <span style={{ background: '#eef', padding: '0.2rem 0.5rem', borderRadius: '0.375rem' }}>email</span> authentication.
+
+![Reset password form](/img/authui/reset_password.png)
+
+You can use the `ResetPasswordForm` component to build your own reset password form.
+
+```tsx title="client/ResetPasswordPage.tsx"
+import { ResetPasswordForm } from '@wasp/auth/forms/ResetPassword'
+
+// Use it like this
+<ResetPasswordForm />
+```
+
+### Verify email form
+
+Useful for <span style={{ background: '#eef', padding: '0.2rem 0.5rem', borderRadius: '0.375rem' }}>email</span> authentication.
+
+![Verify email form](/img/authui/email_verification.png)
+
+You can use the `VerifyEmailForm` component to build your own verify email form.
+
+```tsx title="client/VerifyEmailPage.tsx"
+import { VerifyEmailForm } from '@wasp/auth/forms/VerifyEmail'
+
+// Use it like this
+<VerifyEmailForm />
+```
+
+## Customization
+
+You customize all of the available forms by passing props to them. 
+
+Props you can pass to all of the forms:
+- `appearance` - appearance of the form, see below (optional)
+- `logo` - path to your logo (optional)
+- `socialLayout` - layout of the social buttons, can be `vertical` or `horizontal` (optional)
+
+### Theme colors override
+
+We used [Stitches](https://stitches.dev/) to style the Auth UI. You can customize the styles by overriding the default theme tokens. 
+
+```tsx title="client/LoginPage.tsx"
+import { LoginForm } from '@wasp/auth/forms/Login'
+
+// Define your appearance
+const appearance = {
+  colors: {
+    brand: '#5969b8', // blue
+    brandAccent: '#de5998', // pink
+    submitButtonText: 'white',
+  },
+}
+
+// Use it like this
+<LoginForm appearance={appearance} />
+```
+
+See the list of all available tokens [here](https://github.com/wasp-lang/wasp/blob/main/waspc/data/Generator/templates/react-app/src/stitches.config.js).
```

Comment:
Can you add sth like "more coming soon!" (more tokens)

Reply:
Added